src/Subscriber/PublicationSubscriber.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace CommonGateway\OpenIndex\Subscriber;
+
+use App\Entity\ObjectEntity;
+use App\Entity\Entity;
+use Doctrine\Bundle\DoctrineBundle\EventSubscriber\EventSubscriberInterface;
+use Doctrine\ORM\Events;
+use Doctrine\Persistence\Event\LifecycleEventArgs;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use CommonGateway\CoreBundle\Service\ValidationService;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Subscriber to validate Publication.
+ *
+ * @author Conduction <info@conduction.nl> Barry Brands <barry@conduction.nl>
+ *
+ * @license EUPL <https://github.com/ConductionNL/contactcatalogus/blob/master/LICENSE.md>
+ *
+ * @category Subscriber
+ */
+class PublicationSubscriber implements EventSubscriberInterface
+{
+    const PUBLICATION_REFERENCE = 'https://openwoo.app/schemas/publication.schema.json';
```

Comment:
Shouldn't this be declared public?

Reply:
It isn't used by other services but if thats the preference it can be made public

Reply:
I prefer private unless there is a reason to make it public